ols/app/endpoints/feedback.py

```diff
@@ -102,7 +126,18 @@ def store_user_feedback(
     """
     logger.debug(f"feedback received {feedback_request}")
 
-    user_id = auth[0]
-    store_feedback(user_id, feedback_request.model_dump(exclude=["model_config"]))
+    user_id = retrieve_user_id(auth)
+    try:
+        store_feedback(user_id, feedback_request.model_dump(exclude=["model_config"]))
+    except Exception as e:
+        logger.error("Error storing user feedback")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail={
+                "response": "Error storing user feedback",
+                "cause": str(e),
+            },
+        )
+        logger.error(e)
```

Comment:
@tisnik does this statement actually get executed, given you raised an exception in the previous line?

also why not just log the error as part of the earlier logger.error message?  "error storing feedback: <error message>" seems more helpful.

Reply:
yes, it is leftover, let's update it